tools/serverpod_cli/test/database/migration/defaults/bigint_default_migration_test.dart

```diff
@@ -0,0 +1,134 @@
+import 'package:serverpod_cli/analyzer.dart';
+import 'package:serverpod_cli/src/test_util/builders/database/column_definition_builder.dart';
+import 'package:serverpod_cli/src/test_util/builders/database/database_definition_builder.dart';
+import 'package:serverpod_cli/src/test_util/builders/database/table_definition_builder.dart';
+import 'package:serverpod_service_client/serverpod_service_client.dart';
+import 'package:test/test.dart';
+
+void main() {
```

Comment:
Not sure how useful this test is, as there does not seem to be anything tested from the `BigInt` part, except the the demo string is a "serialized `BigInt`" instead of any random text.

I think these tests only differ for the SQL column type (not Dart type), and thus this test is equivalent to the normal String (== `::text` column) test.

But since we also had this test for `Duration` (which is just a numeric SQL column, and not its own type), I put it in place for now.

Reply:
You are probably right that we in practice already have coverage for this. But I think it makes sense to add for completeness.